assets/js/views/App.vue

```diff
@@ -114,6 +151,40 @@ export default {
 				currency: this.store.state.currency,
 			};
 		},
+		logoutCount() {
+			let count = 0;
+
+			this.providerLogins.forEach(function (login) {
+				if (!login.loggedIn) {
+					count++;
+				}
+			});
+
+			return count;
```

Comment:
Kann man kürzer schreiben:

```suggestion
            return this.providerLogins.filter((login) => !login.loggedIn).length;
```